packages/ssx-sdk/src/ssx.ts

```diff
@@ -112,7 +117,14 @@ export class SSX {
     // get encryption config from config.modules.encryption
     // determine which encryption module to use
     // if encryption module is false, don't initialize encryption or dependent modules
-    this.encryption = new SignatureEncryption({}, this.userAuthorization);
+    if (
+      this.config?.modules?.encryption === true ||
+      (this.config?.modules?.encryption as SSXEncryptionModuleConfig)?.module === 'SignatureEncryption'
+    ) {
+      this.encryption = new SignatureEncryption({}, this.userAuthorization);
+    } else if ((this.config?.modules?.encryption as SSXEncryptionModuleConfig)?.module === 'LitEncryption') {
+      this.encryption = new LitEncryption({}, this.userAuthorization);
+    }
```

Comment:
Just a temporary approach until we define the way we configure the modules. I'm assuming SignatureEncryption as default